src/harness/fourslash.ts

```diff
@@ -2997,30 +2997,18 @@ Actual: ${stringify(fullActual)}`);
             }
         }
 
-        public verifyCodeFixAvailable(negative: boolean, info: FourSlashInterface.VerifyCodeFixAvailableOptions[] | undefined) {
+        public verifyCodeFixAvailable(info: FourSlashInterface.VerifyCodeFixAvailableOptions[]) {
             const codeFixes = this.getCodeFixes(this.activeFile.fileName);
-
-            if (negative) {
-                if (codeFixes.length) {
-                    this.raiseError(`verifyCodeFixAvailable failed - expected no fixes but found ${codeFixes.map(c => c.description)}.`);
+            assert.equal(info.length, codeFixes.length);

```

Comment:
My experience has been that asserts like this tend to produce unhelpful output when they fail.  I haven't found a silver bullet but sometimes I break verification into two steps - the first determines whether or not verification succeeded and the second produces a human-readable representation of the difference between the expected and actual values.
To be more concrete, if you get fewer fixes than expected, I think this will produce a message like "expected 4 but got 3".  It would be much more helpful to know which fixes were actually found or, better still, which were missing.